Dropdown/src/Dropdown.jsx

```diff
@@ -185,16 +185,17 @@ class Dropdown extends Component {
         const index = props.options.findIndex(this.findMatchingItem, this);
         if(index) {
 
-            const value = this.getOption(index).value;
+            const value = this.getOption(index);
 
-            this.setState({
-                closestValue: value,
-                currentIndex: index
-            },() => {
-                this.setState({dropdownText: ""});
-                this.dropdownSearch.value = "";
-                setTimeout(() => this.scrollToSelectedItem(),100);
-            });
+            if(value) {
+                this.setState({
+                    closestValue: value,
+                    currentIndex: index,
+                    dropdownText: ""
+                }, () => {
+                    setTimeout(() => this.scrollToSelectedItem());
```

Comment:
Why is `this.dropdownSearch` not longer updated here?

Reply:
All is managed in "dropdownSearch" method and it was redundant.